core/test/unit/migration_spec.js

```diff
@@ -13,6 +13,11 @@ var should = require('should'), // jshint ignore:line
 
     sandbox = sinon.sandbox.create();
 
+// Clone the fixtures as they get changed during the migration.
+// The initial blog posts will be added a `published_at` property, which
+// would change the fixturesHash.
+fixtures = _.cloneDeep(fixtures);
```

Comment:
I can't reproduce the problem you were running into. The `migration_spec` requires the fixtures and calculates a hash - it's independent from the migration?